docs/source/intro/troubleshooting.rst

```diff
@@ -1,7 +1,321 @@
+
+.. _Stackoverflow link: http://stackoverflow.com/questions/21079820/how-to-find-pg-config-pathlink
+
+Ensuring a UTF-8 locale
+.......................
+
+For some reasons, on some machines (notably often on Mac OS X) there is no default locale defined, and when you run ``verdi setup`` for the first time it fails (see also `this issue`_ of django).
+Run in your terminal (or maybe even better, add to your ``.bashrc``, but then remember to open a new shell window!):
+
+.. code-block:: bash
+
+   export LANG="en_US.UTF-8"
+   export LC_ALL="en_US.UTF-8"
+
+and then run ``verdi setup`` again.
+
+.. _this issue: https://code.djangoproject.com/ticket/16017
+
+Possible Ubuntu dependencies
+.............................
+
+Several users reported the need to install also ``libpq-dev`` (header files for libpq5 - PostgreSQL library):
+
+.. code-block:: console
+
+   $ apt-get install libpq-dev
+
+But under Ubuntu 12.04 this is not needed.
+
+verdi not in PATH
+-----------------
+
+Installing the ``aiida-core`` python package *should* add the ``verdi`` CLI to your ``PATH`` automatically.
+
+If the ``verdi`` executable is not available in your terminal, the folder where ``pip`` places binaries may not be added to your ``PATH``
+
+For Linux systems, this folder is usually something like ``~/.local/bin``:
+
+.. code-block:: bash
+
+   export PATH=~/.local/bin:${PATH}
+
+For Mac OS X systems, the path to add is usually ``~/Library/Python/2.7/bin``:
+
+.. code-block:: bash
+
+   export PATH=~/Library/Python/2.7/bin:${PATH}
+
+After updating your ``PATH``, the ``verdi`` command should be available.
+
+.. note::
+
+   A preprequisite for ``verdi`` to work is that the ``aiida`` python package is importable.
+   Test this by opening a ``python`` or ``ipython`` shell and typing:
+
+   .. code-block:: python
+
+      import aiida
+
+   If you get an ``ImportError`` (and you are in the environment where AiiDA was installed), you can add it to the ``PYTHONPATH`` manually:
+
+   .. code-block:: bash
+
+      export PYTHONPATH="${PYTHONPATH}:<AiiDA_folder>"
+
+
+Configuring remote SSH computers
+--------------------------------
+
+ssh_kerberos installation
+.........................
+
+When installing the ``ssh_kerberos`` *optional* requirement through Anaconda you may encounter the following error on Ubuntu machines:
+
+.. code-block:: console
+
+   version 'GFORTRAN_1.4' not found (required by /usr/lib/libblas.so.3)
+
+This is related to an open issue in anaconda `ContinuumIO/anaconda-issues#686`_.
+A potential solution is to run the following command:
+
+.. code-block:: console
+
+   $ export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libgfortran.so.3
+
+.. _ContinuumIO/anaconda-issues#686: https://github.com/ContinuumIO/anaconda-issues/issues/686
+
+Output from .bashrc and/or .bash_profile on remote computers
+............................................................
+
+..note:: This applies also computers configured via ``local`` transport!
```

Comment:
This does not output correctly.

Reply:
fixed in 26d315b634043048c4ce107c87c8e165e2368a60